include/libtorrent/add_torrent_params.hpp

```diff
@@ -179,6 +179,9 @@ TORRENT_VERSION_NAMESPACE_2
 		// (see torrent_handle::add_extension()).
 		void* userdata = nullptr;
 
+		// The ``clientdata`` parameter is optional and will be stored in the torrent.
```

Comment:
this comment should probably mention how to get it back. e.g. `torrent_handle::clientdata()`, and maybe that it's included in `torrent_removed_alert` too